lib/storage/sqlite/campaign-info-storage.js

```diff
@@ -53,4 +53,22 @@ class CampaignInfoStorage {
   }
 }
 
+/**
+ * Fetch a CampaignInfoByCandidate object by id
+ *
+ * @param {Number} id candidate_id
+ * @param {campaignInfoProcessor} cb
+ */
+CampaignInfoStorage.prototype.fetchByCampaign = function (id, cb) {
+  db.open(this.dbpath)
+    .then((stmt) => stmt.all('SELECT * FROM campaign_info WHERE candidate_id = ?', id))
```

Comment:
What you're actually implementing by fetching and returning this data is the `/candidates/:id/campaigns/info` endpoint, since you're returning an array of [CampaignInfo](https://github.com/Lever-age/api/wiki/Interface#campaign-info) objects. The `/candidates/:id/campaigns` endpoint calls for returning an array of [Campaign](https://github.com/Lever-age/api/wiki/Interface#campaign) objects, which are a touch more complex (You can see an example of building that object in [the candidate storage method I've written but not merged yet](https://github.com/lottspot/leverage-api/blob/endpoint/candidate/lib/storage/sqlite/candidate-storage.js)). Probably the simplest thing to do here is just change the endpoint to `/candidates/:id/campaigns/info`, since the work is done for literally exactly that